demos/collection/table2.php

```diff
@@ -22,7 +22,7 @@
 $table->template->appendHTML('Body', '<tr class="center aligned"><td colspan=2>This is part of body, goes before other rows</td></tr>');
 
 // Hook can be used to display data before row. You can also inject and format extra rows.
-$table->onHook('beforeRow', function (atk4\ui\Table $table) {
+$table->onHook(\atk4\ui\Lister::HOOK_BEFORE_ROW, function (atk4\ui\Table $table) {
```

Comment:
This is confusing even if correct. Previously it was a lot easier to just use hook name, but now developer should also know which class to use for hook constant :(
Maybe could use `\atk4\ui\Table::HOOK_BEFORE_ROW` here instead? Can these constants be used in extended classes?